subgraphs/openzeppelin-governor/protocols/code4rena-governance/config/templates/code4rena-governance.template.yaml

```diff
@@ -0,0 +1,79 @@
+specVersion: 0.0.2
+repository: https://github.com/messari/subgraphs
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: ArenaGovernor
+    network: {{ network }}
+    source:
+      address: "{{ governorContractAddress }}"
+      abi: ArenaGovernor
+      startBlock: {{ governorContractStartBlock }}
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      entities:
+        - Governance
+        - Proposal
+        - Vote
+        - TokenHolder
+        - Delegate
+      abis:
+        - name: ArenaGovernor
+          file: ./abis/code4rena-governance/ArenaGovernor.json
+      eventHandlers:
+        # Part of GovernorPreventLateQuorum 
+        # Our entities currently don't track deadline
+        # - event: LateQuorumVoteExtensionSet(uint64,uint64)
```

Comment:
I'd be curious to know what fields are involved in this event and how they would effect the proposal entity fields

Feels like we should be handling it.. In a universe where we have a robust notification system (soon) this feels like a case that would be of importance to us

Reply:
Goes for the below `ProposalExtended` event as well

Reply:
`LateQuorumVoteExtensionSet` changes the default number of blocks for an extension. imo it doesnt really serve any purpose to track.

But on second looks, i realise we can support `ProposalExtended` simply be updating the `endBlock`

Reply:
added tiny commit with extra handler for ProposalExtended